dev/com.ibm.ws.jca.cm/src/com/ibm/ejs/j2c/PoolManagerMBeanImpl.java

```diff
@@ -104,16 +104,28 @@ public void unregister() {
     @Override
     public Object invoke(String actionName, Object[] params, String[] signature) throws MBeanException, ReflectionException {
         if (TraceComponent.isAnyTracingEnabled() && tc.isEntryEnabled())
-            Tr.entry(this, tc, "invoke");
+            Tr.entry(this, tc, "invoke", params, signature);
```

Comment:
I think it would be good to trace the `actionName` here at some point also, otherwise we won't get the method name

Reply:
The actionName is covered by an exception, which is easy to debug,  throw new MBeanException(new IllegalArgumentException(actionName), Tr.formatMessage(tc, "INVALID_MBEAN_INVOKE_ACTION_J2CA8061", actionName, "ConnectionManager"));